projects/msvc/mupen64plus-core.vcxproj

```diff
@@ -677,13 +677,13 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='New_Dynarec_Release|Win32'">true</ExcludedFromBuild>
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='New_Dynarec_Release|x64'">false</ExcludedFromBuild>
       <FileType>Document</FileType>
-      <Command Condition="'$(Configuration)|$(Platform)'=='New_Dynarec_Debug|x64'">"..\..\..\mupen64plus-win32-deps\nasm-2.14.02\nasm.exe" -i ..\..\src\ -i ..\..\src\asm_defines\ -o $(IntDir)linkage_x64.obj -f win64 -d WIN64 %(FullPath)</Command>
+      <Command Condition="'$(Configuration)|$(Platform)'=='New_Dynarec_Debug|x64'">"..\..\..\mupen64plus-win32-deps\nasm-2.14.02\x64\nasm.exe" -i ..\..\src\ -i ..\..\src\asm_defines\ -o $(IntDir)linkage_x64.obj -f win64 -d WIN64 %(FullPath)</Command>
```

Comment:
Where `x64` sub directory come from ?

https://github.com/mupen64plus/mupen64plus-win32-deps/tree/master/nasm-2.14.02

Looks like it’s not here.

Reply:
I just realize you download `nasm-2.14.02-win32.zip` and `nasm-2.14.02-win64.zip` and extract them in their respective directory.

Shouldn’t we add those to win32 repo ? https://github.com/mupen64plus/mupen64plus-win32-deps

Reply:
Indeed, I will make the appropriate arranges once the dependencies are in place.